### PR TITLE
Update Chat.java

### PR DESCRIPTION
**Pull request by juliachin123**
Updated how the line is parsed in handleCommand

### CODE REVIEW

**Review by evelez7 — COMMENTED** [review]
LGTM

**Comment by netcables** [comment]
LGTM

**Review by netcables — COMMENTED** [review]
LGTM

File: src/codeu/chat/client/commandline/Chat.java [hunk 1]
```diff
@@ -48,9 +48,13 @@ public Chat(Context context) {
   //
   public boolean handleCommand(String line) {
 
-    final Scanner tokens = new Scanner(line.trim());
-
-    final String command = tokens.hasNext() ? tokens.next() : "";
```
Comment (Tuxdude) [inline src/codeu/chat/client/commandline/Chat.java:53]: Have you tried compiling the code with these changes? You have removed `tokens` variable which is still being used in line 74. Am I missing something?

**Review by Tuxdude — COMMENTED** [review]
Please verify if the code compiles with these changes.

**Pull request closed without merging**